src/appsscript.json

```diff
@@ -29,5 +33,9 @@
   "urlFetchWhitelist": [
     "https://api.data.world/v0/",
     "https://data.world/oauth"
-  ]
-}
+  ],
+  "webapp": {
+    "executeAs": "USER_DEPLOYING",
+    "access": "MYSELF"
+  }
```

Comment:
This may be a side-effect of how I had to deploy this on my own during testing, the UI required that I select a Project Option and `webapp` was my only option. I will be able to confirm once I get access to our production deployment of the Data Studio Connector.

Reply:
Does this mean that this may not be required if we are deploying this to production?